test/e2e/framework/util.go

```diff
@@ -163,6 +163,9 @@ const (
 
 	// Number of times we want to retry Updates in case of conflict
 	UpdateRetries = 5
+
+	// number of objects that gc can delete in a second
+	gcThroughput = 15
```

Comment:
@caesarxuchao Did you use default QPS limits in density? In Jenkins we have them increased, which means that it's more likely to pass in Jenkins.
BTW - it seems that indeed it is passing in 500-gce density, but it's not passing in 2000-gce density.

@gmarek - I also don't know where 15 comes from... :) I would suggest 10 or sth like (with some explanatory comment like what @caesarxuchao wrote above).